samples/recognize.v1p1beta1.js

```diff
@@ -33,6 +33,11 @@ function syncRecognizeModelSelection(
   // [START speech_transcribe_model_selection]
   // Imports the Google Cloud client library
   const fs = require('fs');
```

Comment:
While you're doing this, could you remove the existing `// Imports the Google Cloud client library` lines?  🙏

Reply:
I kept that comment above `const fs = require('fs');` since this one was not a beta import. I believe we should not remove it. No?